packages/bpk-component-image/src/__snapshots__/BpkImage-test.js.snap

```diff
@@ -545,3 +545,107 @@ exports[`BpkImage should render correctly 1`] = `
   </div>
 </div>
 `;
+
+exports[`BpkImage should support srcSet 1`] = `
```

Comment:
this snapshot doesn't assert against any sort of `srcset` support

Reply:
Ah - `inView` was set to false!